Sources/HaCWebsiteLib/Models/Workshop+init.swift

```diff
@@ -39,6 +40,7 @@ extension Workshop {
     slidesLink = try builder.getSlidesLink()
     tags = try builder.getTags()
     license = try builder.getLicense()
+    hasLicense = try Workshop.getHasLicense(localPath: localPath)
```

Comment:
This license / hasLicense thing looks a bit confusing.
Could you explain again the justification for having both the license ID in the metadata.yaml _and_ the LICENSE text file? Maybe we should get rid of the license id

Alternatively, why not rename the `license` field on Workshop to `licenseId`